drivers/hmis/app/models/hmis/form/definition.rb

```diff
@@ -413,6 +413,15 @@ def valid_status_for_submit?
     published? || retired?
   end
 
+  def supports_save_in_progress?
+    walk_definition_nodes(as_open_struct: true) do |item|
+      # Saving assessments with File item types isn't supported
+      return false if item.type == 'FILE'
```

Comment:
is there any benefit to calling `.upcase` here to ensure the casing is the same, or are we confident that `.type` will always be upcased?

Reply:
I think we are confident that it will always be uppercase, because otherwise it would be an invalid form definition ([schema here](https://github.com/greenriver/hmis-warehouse/blob/681544cdafe35c9ec96ccc0e706a14cb371382a3/drivers/hmis_external_apis/public/schemas/form_definition.json#L4)).